src/JSONErrorHandlerMiddleware.ts

```diff
@@ -45,6 +54,7 @@ export class JSONErrorHandlerMiddleware
         body: JSON.stringify(omit(['stack'], serializeError(error))),
         statusCode: error.statusCode
       }
+      addContentTypeHeader(request.response)
```

Comment:
I think the logic for adding the header can be simplified to
```suggestion
        headers: {
          'Content-Type': 'application/json'
        }
      }
```